opal/mca/common/cuda/common_cuda.c

```diff
@@ -300,6 +303,15 @@ void mca_common_cuda_register_mca_variables(void)
                                  MCA_BASE_VAR_SCOPE_READONLY,
                                  &mca_common_cuda_cumemcpy_timing);
 #endif /* OPAL_ENABLE_DEBUG */
+
+#if OPAL_CUDA_GPU_MEM_WAR
```

Comment:
Yes, I see.  My thought is eventually, this code will be removed, and by putting the #ifdef around it, it happens naturally.  But I will do as you suggest and then just remove it someday when I can.